packages/wonder-blocks-dropdown/components/dropdown-core.js

```diff
@@ -490,6 +558,7 @@ export default class DropdownCore extends React.Component<
                     dropdownStyle,
                 ]}
             >
+                {displaySearchTextInput && this.renderSearchTextInput()}
```

Comment:
Don't we want the filter to appear in the opener?

Reply:
I had another look at the figma and you're right.  I posted a comment requesting mocks of the dropdown with filter search field and the opener.  I think that help give us a better idea of what this thing should look like when it's finished. 

I posted another comment ini the figma asking about scroll behavior within the dropdown.  Right now it looks like the search field will scroll out of the view as soon as someone starts scrolling down.  I'm not sure that's the behavior we want because then people are forced to scroll up to the top of the list if they want to filter the list.